0.15.0-rc.3 doesn't build for wasm when feature custom_cursor is not enabled

## Bevy version

0.15.0-rc.3 and main

## \[Optional\] Relevant system information

Building for wasm32-unknown-unknown:

## What you did

Build for wasm32-unknown-unknown without default features, but still with windowing and a few things enabled

Easiest way to reproduce is `cargo check --target wasm32-unknown-unknown -p bevy_winit`

## What went wrong


```
    Checking bevy_winit v0.15.0-rc.3 (bevy/crates/bevy_winit)
error[E0433]: failed to resolve: use of undeclared type `CustomCursor`
   --> crates/bevy_winit/src/cursor.rs:158:32
    |
158 |             CursorIcon::Custom(CustomCursor::Url { url, hotspot }) => {
    |                                ^^^^^^^^^^^^ use of undeclared type `CustomCursor`
    |
help: consider importing this struct
    |
3   + use winit::window::CustomCursor;
    |

error[E0425]: cannot find value `cursor_cache` in this scope
   --> crates/bevy_winit/src/cursor.rs:161:20
    |
161 |                 if cursor_cache.0.contains_key(&cache_key) {
    |                    ^^^^^^^^^^^^ not found in this scope

error[E0433]: failed to resolve: use of undeclared type `WinitCustomCursor`
   --> crates/bevy_winit/src/cursor.rs:165:34
    |
165 |                     let source = WinitCustomCursor::from_url(url.clone(), hotspot.0, hotspot.1);
    |                                  ^^^^^^^^^^^^^^^^^ use of undeclared type `WinitCustomCursor`
    |
help: consider importing this struct through its public re-export
    |
3   + use crate::WinitCustomCursor;
    |

warning: unused import: `crate::CustomCursorExtWebSys`
   --> crates/bevy_winit/src/cursor.rs:164:25
    |
164 |                     use crate::CustomCursorExtWebSys;
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(unused_imports)]` on by default

error[E0599]: no variant or associated item named `Custom` found for enum `cursor::CursorIcon` in the current scope
   --> crates/bevy_winit/src/cursor.rs:158:25
    |
54  | pub enum CursorIcon {
    | ------------------- variant or associated item `Custom` not found for this enum
...
158 |             CursorIcon::Custom(CustomCursor::Url { url, hotspot }) => {
    |                         ^^^^^^ variant or associated item not found in `CursorIcon`

error[E0433]: failed to resolve: use of undeclared type `CustomCursorCacheKey`
   --> crates/bevy_winit/src/cursor.rs:159:33
    |
159 |                 let cache_key = CustomCursorCacheKey::Url(url.clone());
    |                                 ^^^^^^^^^^^^^^^^^^^^ use of undeclared type `CustomCursorCacheKey`

error[E0599]: no variant or associated item named `CustomCached` found for enum `CursorSource` in the current scope
   --> crates/bevy_winit/src/cursor.rs:162:35
    |
162 |                     CursorSource::CustomCached(cache_key)
    |                                   ^^^^^^^^^^^^ variant or associated item not found in `CursorSource`
    |
   ::: crates/bevy_winit/src/state.rs:163:1
    |
163 | pub enum CursorSource {
    | --------------------- variant or associated item `CustomCached` not found for this enum

error[E0599]: no variant or associated item named `Custom` found for enum `CursorSource` in the current scope
   --> crates/bevy_winit/src/cursor.rs:166:35
    |
166 |                     CursorSource::Custom((cache_key, source))
    |                                   ^^^^^^ variant or associated item not found in `CursorSource`
    |
   ::: crates/bevy_winit/src/state.rs:163:1
    |
163 | pub enum CursorSource {
    | --------------------- variant or associated item `Custom` not found for this enum
```

--- a/crates/bevy_winit/src/cursor.rs
+++ b/crates/bevy_winit/src/cursor.rs
@@ -154,7 +154,11 @@ fn update_cursors(
                     CursorSource::Custom((cache_key, source))
                 }
             }
-            #[cfg(all(target_family = "wasm", target_os = "unknown"))]
+            #[cfg(all(
+                feature = "custom_cursor",
+                target_family = "wasm",
+                target_os = "unknown"
+            ))]
             CursorIcon::Custom(CustomCursor::Url { url, hotspot }) => {
                 let cache_key = CustomCursorCacheKey::Url(url.clone());
 

